### PR TITLE
Attempting to fix timeout during E2E tests

### PR DESCRIPTION
**Pull request by ncksllvn**

### CODE REVIEW

File: test/discharge-wizard/00-required.e2e.spec.js [hunk 1]
```diff
@@ -42,12 +42,11 @@ module.exports = E2eHelpers.createE2eTest(
...
 
-    client.pause(Timeouts.verySlow);
-
     // results page
     client
       .click('a.usa-button-primary')
-      .waitForElementVisible('.dw-guidance', Timeouts.verySlow)
+      .waitForElementVisible('.dw-guidance', Timeouts.slow)
+      .pause(Timeouts.verySlow)
       .axeCheck('.main');
```
Comment (ncksllvn) [inline test/discharge-wizard/00-required.e2e.spec.js]: There's a PDF download occurring here to `'http://arba.army.pentagon.mil/documents/dd0293.pdf'`. It's too fast during local testing to register, but I think Jenkin's is stumbling on it.

**Review by bshyong — APPROVED** [review]
LGTM

File: test/discharge-wizard/00-required.e2e.spec.js [hunk 2]
```diff
@@ -42,12 +42,9 @@ module.exports = E2eHelpers.createE2eTest(
       .axeCheck('.main')
       .click('a.usa-button-primary');
 
-    client.pause(Timeouts.verySlow);
-
     // results page
     client
-      .click('a.usa-button-primary')
-      .waitForElementVisible('.dw-guidance', Timeouts.verySlow)
+      .waitForElementVisible('.dw-guidance', Timeouts.slow)
```
Comment (jbalboni) [inline test/discharge-wizard/00-required.e2e.spec.js]: It seems like this removes a button click. Is that intentional?

**Review by jbalboni — APPROVED** [review]
LGTM, had one question.

File: test/discharge-wizard/00-required.e2e.spec.js (hunk 2 above)
Comment (ncksllvn) [reply test/discharge-wizard/00-required.e2e.spec.js]: Yeah it does. That click was unintentionally in there twice and the second click was causing an error.

**Pull request merged**